cmd/iavlserver/main.go

```diff
@@ -0,0 +1,191 @@
+package main
```

Comment:
I keep getting these log messages on all HTTP requests:

```
Failed to extract ServerMetadata from context
```

Reply:
I think that's the equivalent of a 404, I don't know how to make it more precise but I only notice that when sending a message to an endpoint that doesn't exist.